src/test/java/SeleniumFramework.java

```diff
@@ -1,51 +1,72 @@
+        WebElement login_button = driver.findElement(By.cssSelector("a[href*='login']"));
+        login_button.click();
+        login_button.sendKeys("Mendygax@gmail.com");
+        Thread.sleep(1000);
+
+        WebElement password = driver.findElement(By.cssSelector("#password"));
+        password.click();
+        password.sendKeys("Positive89");
+
+        Thread.sleep(1000);
+
+        driver.findElement(By.cssSelector("[data-testid=\'loginSubmit\']")).click();
+
+        Thread.sleep(2000);
+        driver.quit();
+    }
+
+//    TC2 Login Wrong credentials (not existing email)
+    @Test
+    public static void WrongEmail() throws InterruptedException {
+        WebDriver driver = new ChromeDriver();
+        driver.get("https://deens.com/?internaltraffic");
 
-        WebElement login_button = driver.findElement(By.cssSelector("[data-testid=\'loginSubmit\']"));
+        WebElement login_button = driver.findElement(By.cssSelector("a[href*='login']"));
         login_button.click();
+        login_button.sendKeys("mendy@gmail.com");
+        Thread.sleep(1000);
 
-        WebElement uiErrorMessage = driver.findElement(By.cssSelector(".ui.error.message p"));
+        WebElement password = driver.findElement(By.cssSelector("#password"));
+        password.click();
+        password.sendKeys("Positive89");
 
-        String errorMessagetext = uiErrorMessage.getText();
-        System.out.println(errorMessagetext);
+        Thread.sleep(1000);
 
-        Assert.assertEquals(errorMessagetext, "Empty email or password");
+        driver.findElement(By.cssSelector("[data-testid=\'loginSubmit\']")).click();
 
+        Thread.sleep(2000);
 
+        Assert.assertTrue(driver.findElement(By.cssSelector("[class=\'ui error message\']")).isDisplayed());
+        driver.quit();
     }
-}
-
-//    @Test
-//    public void IncorrectEmailAndPassword() {
-//        WebDriver driver = new ChromeDriver();
-//        driver.manage().timeouts().implicitlyWait(10, TimeUnit.SECONDS);
-//
-//        WebElement emailField = driver.findElement(By.cssSelector("#email"));
-//        emailField.sendKeys("incorrect@email.com");
-//
-//        WebElement passwordField = driver.findElement(By.cssSelector("password"));
-//        passwordField.sendKeys("incorrect password");
-//
-//        WebElement login_button = driver.findElement(By.cssSelector("[data-testid=\'loginSubmit\']"));
-//        login_button.click();
-//
-//        WebElement uiErrorMessage = driver.findElement(By.cssSelector(".ui.error.message p"));
-//
-//        String errorMessagetext = uiErrorMessage.getText();
-//        System.out.println(errorMessagetext);
-//
-//        Assert.assertEquals(errorMessagetext, "Empty email or password");
+
+//    TC3 Login Empty Credentials
+    @Test
+    public static void EmptyCredentials() throws InterruptedException {
+        WebDriver driver = new ChromeDriver();
+        driver.get("https://deens.com/?internaltraffic");
+
```

Comment:
Please use maximaze method for driver window or handle this element which I mentioned above